datafusion/src/physical_plan/expressions/cast.rs

```diff
@@ -95,6 +95,24 @@ impl PhysicalExpr for CastExpr {
     }
 }
 
+impl CastExpr {
+    pub(crate) fn cast_scalar_default_options(
+        scalar: ScalarValue,
+        cast_type: &DataType,
+    ) -> Result<ScalarValue> {
+        CastExpr::cast_scalar_type(scalar, cast_type, &DEFAULT_DATAFUSION_CAST_OPTIONS)
+    }
+    pub(crate) fn cast_scalar_type(
```

Comment:
nitpick, perhaps these can be implemented as simple pub(crate) functions like the `cast_column` function below? Any reason why we want to make it a struct method?